src/Core/Components/DateTime/FluentDatePicker.razor.cs

```diff
@@ -50,6 +50,18 @@ protected override string? ClassValue
     [Parameter]
     public virtual EventCallback<DateTime> PickerMonthChanged { get; set; }
 
+    /// <summary>
+    /// Command executed when the user double-clicks on the date picker.
+    /// </summary>
+    [Parameter]
+    public EventCallback<MouseEventArgs> OnDoubleClick { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether today's date is set when double-clicking on the text field of date picker.
+    /// </summary>
+    [Parameter]
+    public bool DoubleClickToToday { get; set; }
```

Comment:
As explain here: https://github.com/microsoft/fluentui-blazor/discussions/2524#discussioncomment-10423382

I think the property should be `DateTime DoubleClickToDate` which allows you to specify a very specific date or the current date `DateTime.Today`. This could help resolve other cases.

Reply:
The `DoubleClickToToday` has been changed to `DoubleClickToDate`.